spacemacs/search-symbol.vim

```diff
@@ -14,7 +14,7 @@ vnoremap <leader>sh    <Esc>:action HighlightUsagesInFile<CR>
 nnoremap <leader>sP    :action FindUsages<CR>
 vnoremap <leader>sP    :action FindUsages<CR>
 
-" Search in project. Unfortunately <leader>/ does not work.
```

Comment:
This seems to be working for me.  Do you want to give it a shot and see if it works for you?

Reply:
it works for me, too, great!